packages/proxyIdentity/contracts/IdentityManager.sol

```diff
@@ -1,12 +1,40 @@
-pragma solidity 0.8.0;
+pragma solidity 0.4.26;
```

Comment:
Is the solidity downgrade so that we can use the optionality CloneFactory?

Reply:
Yes, optionality restricts soliditiy by v0.4